sql/dj_setup_database.in

```diff
@@ -34,6 +35,8 @@ Commands:
   install-examples install examples only
   install-loadtest configure for load testing. WARNING: CREATES A LOT OF EXTRA ITEMS!
   upgrade          upgrade MySQL database schema to current version
+  dump             backup the current database to file (without .tar.gz suffix)
+  load             load a backup from file (without.tar.gz suffix). REMOVES ALL PREVIOUS DATA!
```

Comment:
It would be good to explicitly document here that dump and load have an (optional) file as argument. And then add on the `Usage` line: ` [argument]...`